src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -99,6 +100,219 @@ public async Task<List<Rule>> TryWriteDelegationPolicyRules(List<Rule> rules)
 
             return result;
         }
+                
+        private async Task<List<Rule>> ProcessPolicyFile(string policyPath, string org, string app, RequestToDelete deleteRequest)
```

Comment:
There is no order in public and private methods in this class. Surprised there's not a style cop/build warning for this

Reply:
I did think to fix this but wantet to develope those as near to usage scroll less but forgot to remove as it did not create a new code smell. Will Fix.

Reply:
Fixed